WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -3,12 +3,14 @@ import UIKit
 import RxSwift
 import WordPressComAnalytics
 
-func AccountSettingsViewController(account account: WPAccount) -> ImmuTableViewController {
-    let service = AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+func AccountSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    let service = account.map({ account in
+        return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+    })
```

Comment:
Just a thought on this one. Since `account` is an optional parameter, and not a collection, wouldn't be more appropriate to use... `if let` / `if guard` ?.

Plus ... if the account is *nil*, which shouldn't happen, but apparently, would... wouldn't the rest of the code break?